### ISSUE TITLE
Search page *needs* help

### ISSUE THREAD

**Issue by SoftwareMaven** [django/djangopeople#26]
Given the map on the front page, I thought it would be interesting to see other Django guys near me, so I hit "Search". All that shows up is a text field, no description, no help, nothing. Now I don't know what to search for, but I figure if there is a map on the front page, the search should be map based, so I enter a zip code.

And now, I know that it only searches user names and real names. It was nice enough to point out I was using it wrong, but not nice enough to point out how to do it right.

TL;DR: Please put instructions on the search page.
